src/OSPSuite.Presentation/Presenters/Charts/DataBrowserPresenter.cs

```diff
@@ -155,6 +168,29 @@ public void UsedChangedFor(DataColumnDTO dataColumnDTO, bool used)
       {
          raiseUsedChanged(dataColumnDTO.DataColumn, used);
          updateDataSelection(_view.SelectedColumns);
+         updateLinkedObservedData(dataColumnDTO.DataColumn, used);
+      }
+
+      private void updateLinkedObservedData(DataColumn dataColumn, bool used)
+      {
+         if (!_isLinkedMappedOutputs) return;
+         
+         var linkedObservedData = getLinkedObservedDataFromOutputPath(dataColumn.PathAsString);
+         SetUsedState(linkedObservedData, used);
+      }
+
+      private List<DataColumnDTO> getLinkedObservedDataFromOutputPath(string updatedDataColumnName)
+      {
+         var linkedObservedDataRepositories = AllOutputMappings.AllDataRepositoryMappedTo(updatedDataColumnName);
+         var linkedObservedData = getDataColumnDTOsFromDatarepositories(linkedObservedDataRepositories);
+         return linkedObservedData;
+      }
+
+      private List<DataColumnDTO> getDataColumnDTOsFromDatarepositories(IEnumerable<DataRepository> linkedObservedDataRepositories)
```

Comment:
getDataColumnDTOsFromDatarepositories => getDataColumnDTOsFromDataRepositories

(use resharper for this)
![image](https://user-images.githubusercontent.com/1041237/177205446-2de4f8eb-515e-4510-bf52-7ed85be89b9d.png)

Reply:
typo fixed